diesel_cli/src/main.rs

```diff
@@ -606,6 +607,31 @@ fn regenerate_schema_if_file_specified(
     Ok(())
 }
 
+// List which features have been compiled into this version of diesel_cli
+#[cfg(any(feature = "postgres", feature = "mysql", feature = "sqlite"))]
+fn show_which_engines_supported() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let mut features: Vec<&str> = Vec::new();
+    #[cfg(feature = "postgres")]
+    features.push("postgres");
+    #[cfg(feature = "mysql")]
+    features.push("mysql");
+    #[cfg(feature = "sqlite")]
+    features.push("sqlite");
+
+    for f in &features {
+        match f as &str {
+            "postgres" | "sqlite" | "mysql" | "sqlite-bundled" => print!("{} ", f),
+            other => print!("'{}' ", other),
```

Comment:
Not sure why we need this.

Reply:
In the spirit for rust's match I though that this would provide completeness while also providing a method of highlighting any future features that were implemented and enabled.

Reply:
Just because there's `match` in rust doesn't mean we have to use it. What does this subcommand need to do? Print all the features that were pushed into the `features` vec. Why do we care about `other`? It's not someone else giving us this `features` value. It's us who are building it.